addons/mail/static/src/models/messaging/messaging.js

```diff
@@ -340,6 +340,14 @@ registerModel({
             isCausal: true,
             readonly: true,
         }),
+        minuteClock: one('Clock', {
+            default: insertAndReplace({
+                frequency: 60 * 1000,
+            }),
+            isCausal: true,
+            readonly: true,
+            required: true,
+        }),
```

Comment:
Not really needed as a field on messaging. Could just use `insertAndReplace({ frequency: 60 * 1000 }),` where it is needed, and it will be created if missing, and re-used otherwise.